mock-core-configs/etc/mock/templates/openeuler-22.03.tpl

```diff
@@ -35,40 +35,47 @@ name=OS
 baseurl=http://repo.openeuler.org/openEuler-22.03-LTS-SP1/OS/$basearch/
 enabled=1
 gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/openeuler/RPM-GPG-KEY-openEuler
+gpgkey=http://repo.openeuler.org/openEuler-22.03-LTS-SP1/OS/$basearch/RPM-GPG-KEY-openEuler
```

Comment:
GPG keys cannot be remote for Mock. Please add the new key to https://github.com/xsuchy/distribution-gpg-keys

Reply:
Ok, i will try to make a PR.

Reply:
@Conan-Kudo As the PR merged, should we make the next release mock-core-configs depends on the newest distribution-gpg-keys?

Reply:
Once this PR is merged, I'll wrap a new mock-core-config release :)